cmd/invoke.go

```diff
@@ -62,9 +62,15 @@ dapr invoke --unix-domain-socket --app-id target --method sample --verb GET
 		client := standalone.NewClient()
 
 		// TODO(@daixiang0): add Windows support
-		if runtime.GOOS == "windows" && invokeSocket != "" {
-			print.FailureStatusEvent(os.Stderr, "unix-domain-socket option still does not support Windows!")
-			os.Exit(1)
+		if unixDomainSocket != "" {
```

Comment:
unixDomainSocket is name of variable that is specifically bound for `dapr run` command. See line https://github.com/dapr/cli/pull/822/files#diff-8146f8148ccbf6711d65f532f6ab9a7c8dfbdc3960c7ffc974f17d6d224dd349L333

For invoke it is invokeSocket and for publish it is publishSocket ... 
https://github.com/dapr/cli/pull/822/files#diff-b52b35cfbab321cb5e86cfb5c808b50bcba38f6cbac5950d915689f830a40c78L91

https://github.com/dapr/cli/pull/822/files#diff-780c49c8baa9543ca451bf8d3fa2cd45f4531c9043de95ab7e43d5aa1d573cddL79